src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -0,0 +1,486 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "Pane.h"
+
+using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Core;
+using namespace winrt::Microsoft::Terminal::Settings;
+using namespace winrt::Microsoft::Terminal::TerminalControl;
+
+static const int SEPERATOR_SIZE = 4;
+
+Pane::Pane(GUID profile, TermControl control, const bool lastFocused) :
+    _control{ control },

```

Comment:
does cppwinrt support std::move? can we std::move these winrt components to avoid a needless addref/release?

Reply:
or the constructor has to take in a `const TermControl&`

Reply:
I honestly don't know if it supports some sort of move - @kennykerr do you know the best way to do this?

Reply:
Ah okay I see what's going on here now. Yea these are all const refs now.

Reply:
Yes, all C++/WinRT types can be safely and efficiently moved.

Reply:
Certainly better to use const& wherever possible. Just avoid them for async (e.g. coroutine) functions for obvious reasons.